vscode-extension/package.json

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
We should use a maven plugin to hook this to the Maven build. E.g. https://github.com/oracle/helidon-site/blob/master/pom.xml#L168
If we wrap this with a pom.xml we can pick the LICENSE.txt from the top-level, see https://github.com/oracle/helidon/blob/master/pom.xml#L697 ; there is a trick though as if we let the resources plugin scan from the top level directory it slows down the build significantly. We should fix this the same way as in Helidon by isolating the text files under licensing/